### ISSUE TITLE
/var/lib/dpkg/info/vernemq.postinst: line 11: syntax error near unexpected token `else'

### ISSUE THREAD

**Issue by halfgaar** [vernemq/vernemq#1861]
```
# dpkg -i vernemq-1.12.1.focal.x86_64.deb 
(Reading database ... 110002 files and directories currently installed.)
Preparing to unpack vernemq-1.12.1.focal.x86_64.deb ...
Unpacking vernemq (1626266333:1.12.1) over (1626266333:1.12.1) ...
Setting up vernemq (1626266333:1.12.1) ...
/var/lib/dpkg/info/vernemq.postinst: line 11: syntax error near unexpected token `else'
/var/lib/dpkg/info/vernemq.postinst: line 11: `else'
dpkg: error processing package vernemq (--install):
 installed vernemq package post-installation script subprocess returned error exit status 2
Errors were encountered while processing:
 vernemq
```

Invalid syntax in vernemq.postinst:

```
if ! systemctl is-enabled vernemq >/dev/null 
then
else
    $debsystemctl restart vernemq >/dev/null || true
fi
```

**Comment by ioolkos** [comment]
@halfgaar thanks, solution in the last note in the description here https://github.com/vernemq/vernemq/releases/tag/1.12.1 (I don't know how to make this big and red text)

It's an error in the internal build tools that I haven't been able to fully resolve yet. My apologies.

---
:point_right: Thank you for supporting VerneMQ: https://github.com/sponsors/vernemq
:point_right: Using the binary VerneMQ packages (.deb/.rpm/Docker) requires a subscription.

**Comment by stefanux** [comment]
any updates? this is breaking automation (packages upgrade in the system etc.).

**Comment by ioolkos** [comment]
Fixed in 1.12.2.
